src/objects/Course_Offering__c.object

```diff
@@ -61,6 +57,7 @@
     <enableBulkApi>true</enableBulkApi>
     <enableFeeds>false</enableFeeds>
     <enableHistory>false</enableHistory>
+    <enableLicensing>false</enableLicensing>
```

Comment:
As for this one, it's not in the documentation ???... weird. https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/customobject.htm

Reply:
Yea I've had a hard time tracking this down as well. Don't know if it's a worry (since its false) but we can probably get rid of it.

Reply:
I feel like since its "false" we'd be okay. I cannot find ANY documentation on this either. My guess, in name only -- is that it would be related to being able to enable access to objects via LMA...which we aren't using.